AmplifyPlugins/Auth/AWSAuthPlugin/Dependency/AuthenticationProviderAdapter.swift

```diff
@@ -15,4 +15,8 @@ class AuthenticationProviderAdapter: AuthenticationProviderBehavior {
     init(awsMobileClient: AWSMobileClientBehavior) {
         self.awsMobileClient = awsMobileClient
     }
+
+    func isSignedIn() -> Bool {
+        true //return awsMobileClient.isSignedIn()
```

Comment:
What does this comment mean? Is it a TODO, old behavior that needs to be removed, or what?